File: python/8 kyu/341.py
# https://www.codewars.com/kata/5963c18ecb97be020b0000a2/train/python

# Passed

def derive(coefficient, exponent): 
    result = f"{coefficient * exponent}x^{exponent - 1}"
    return result

output = derive(7, 8)
print(output)

File: nodejs/8 kyu/341.js
// https://www.codewars.com/kata/57ab3c09bb994429df000a4a/train/javascript

// Passed

function twoHighest(arr) {
  arr = Array.from(new Set(arr.sort((a, b) => a - b)));
  let result = [];
  if (arr.length !== 0) {
    result =
      arr.length >= 2 && arr[arr.length - 1] !== arr[0]
        ? [arr[arr.length - 1], arr[arr.length - 2]]
        : [arr[0]];
  }

  return result;
}

const output = twoHighest([15, 20, 20, 17]);
console.log(output);